_posts/2018-04-21-welcome-to-this-week-in-rust.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "Welcome to this week in Rust-WASM! 001"
```

Comment:
I don't think we need every issue to be "welcome", since we already introduced the newsletter in the first issue. TWiR just numbers their issues, and I think that makes sense for us too: https://this-week-in-rust.org/